src/core/evaluator.js

```diff
@@ -1640,19 +1653,37 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                 xObjStateManager.transform(matrix);
               }
 
+              // Enqueue the `textContent` chunk before parsing the /Form
+              // XObject.
+              enqueueChunk();
+              let sinkWrapper = {
+                cachedItems: [],
+                enqueue(chunk, size) {
+                  sink.enqueue(chunk, size);
+                  this.cachedItems.push(chunk.items);
```

Comment:
It's better to have continuous set of items, so `Util.appendToArray(this.cachedItems, chunk.items);` might be better